### ISSUE TITLE
Programmplaner: Sortierung nach Ausstrahlungshäufigkeit

### ISSUE THREAD

**Issue by GWRon** [TVTower/TVTower#1002]
*Dengstar per Discord:*

> Habe jetzt 20 jahre in 2 monaten gespielt u die letzten 5 jahre hatte ich soooviel geld das ich alle filme leergekauft habe (nicht den ramsch) computer hat auch geholfen 🙂 Eine Anordnung nach Ausstrahlungen vermisse ich nur (will gerne immer alle filme gleichoft gezeigt haben,  das is ne fummelige suche 🙂 ) 

An sich eine umsetzbare Sache, aber ich weiss nicht, inwieweit das gebraucht wird.

**Comment by scr0llbaer9000** [comment]
Meine Erfahrungen in dem Kontext:

- Mit dem Datenbestand von vor ein paar Monaten (8.0): Allein die Computer-Gegner kaufen mit der Zeit alle (guten) Filme auf (passiert schon nach 30-40) Tagen, oben gibt es dann nur noch die repetitiven Inhouse-Produktionen. Zeug in der Ramschkiste bleibt ganz gut verfügbar, aber will man ja nicht mehr. Serien gehen sogar komplett aus, der Tisch wird leer, können wohl nicht so schnell nachproduziert werden. Mit dem Update von 8.1 und mehr verfügbaren Filmen wurde das ein wenig mitigiert. Mit den neuen Updates hoffentlich noch mehr.

- Im Programmplaner hätte ich zumindest eine Funktion, dass ich bei einem Film/Programm anzeigen kann, wann ich den zuletzt gesendet habe und dorthin springen kann. Ich muss immer per Sicht suchen, was zeitaufwändig ist.

- Ich versuche nämlich gern, die Filme möglichst zu unterschiedlichen Tageszeiten zu senden, wenn es vom Genre her Sinn macht. Ich finde, das sollte auch ein Balancing-Feature werden, dass der Aktualitätsverschleiß tageszeitabhängig ist. Wenn ich den Prime-Time-Film 24 Stunden später nochmal sende, dann sollte da natürlich große Abnutzung sein. Wenn man aber 12 Stunden später sendet, dann erreicht man potenziell ein Publikum, das man vorher nicht erreichen konnte und die den Film noch nicht gesehen hatten (Schichtarbeiter etc).

**Comment by GWRon** [comment]
Letzteres ist bereits in Teilen drin. Die Aktualitaet sinkt in Abhängigkeit von den erreichten Zuschauerzahlen.

**Comment by nittka** [comment]
Die Sortierung nach Ausstrahlungshäufigkeit im Planer kann ich mir anschauen (ist ja auch optisch noch Platz für einen weiteren Sortierknopf). Halte ich auch für eine interessante Variante: neu gekaufte/neuere Sachen oben.

Je nach Verfügbarkeit der Daten wäre die Anzeige, wann eine Sendung zuletzt ausgestrahlt wurde durchaus möglich. Für die Navigation zu dem Tag fehlt mir erstmal eine Idee, wie das umgesetzt werden kann (beim Hover über der Kassette wird das Datenblatt angezeigt, geht aber weg, wenn man die Maus woanders hinbewegt...) Den Wunsch nach Anzeige der besten Quote statt der größten Zuschauerzahl ("sinnlos" bei Sendegebietserweiterung) gibt es ja auch bereits. Interessant wäre wohl eine Statistik zu einem Programm (Ausstrahlungstag + Zuschauerwerte im Überblick).

Die Abnutzung tageszeitabhängig zu machen, halte ich aktuell für Overkill. Natürlich gibt es zu unterschiedlichen Zeiten unterschiedliches Publikum, aber wenn man das Fass aufmacht, müsste man auch Wochentage, Ferienzeiten, Feiertage, neue Zuschauer im erweiterten Sendegebiet, die einen Film bislang noch nicht gesehen haben, etc. berückschtigen...
"Wenn ein Schichtarbeiter einen Film schauen wollte, hat er einfach den Videorekorder programmiert und wurde zum eigentlichen Ausstrahlungstermin schon mitgezählt." Ich würde argumentieren, dass das schon durch die "erreichbaren Zuschauer", deren Zahl ja auch nicht immer gleich ist, genügend berücksichtigt ist.

**Comment by GWRon** [comment]
Wie gesagt..die Abnutzung ist ja schon abhaengig von der erreichten Zuschauerzahl (primetime vs Nachtwiederholung).


Bezueglich handling von "zum Programm springen". Wenn es nur diese Interaktion gaebe, koennte man rechts neben den Kassetten einen kleinen "Pfeil" einfuehren.

Wenn mehr gewuenscht wuerde, koennte man so ein Hamburgericon rechtseitig einfuehren...was dann die Optionen bereitstellt. "Zur letzten Programmierung", "Quotenstatistiken der letzten Ausstrahlungen"... Sowas halt.

**Comment by nittka** [comment]
Abnutzung: wenn ich es richtig verstanden haben, sollte ja trotz Ausstrahlung zur Primetime (mit voller Abnutzung), eine Ausstrahlung um 10 eine höhere Quote geben, da es da andere Zuschauer gibt, die zur Primetime gar nicht hätten schauen können.

**Comment by GWRon** [comment]
Ja, dem hab ich ja schon indirekt eine Absage erteilt. Sowas laesst sich nur dann halbwegs durchfuehren, wenn vlt 5-8000 Personen "simuliert" wuerden und man anhand derer abspeichert, wer was gesehen hat. Und selbst das waere ungenau.

Alternative waere das fuer den eigenen Spielermarkt (Sendegebiet) zu machen und bei Aenderungen zu skalieren. Durch Verkauf und sofortigen Neukauf an gleicher Stelle koennte man aber entsprechend die Leute dort "vergessen lassen".
